docs/access/vscode.md

```diff
@@ -57,13 +57,15 @@ run the `code` executable that you downloaded the `tunnel` argument.
 You will be asked to choose whether to log in to Microsoft or GitHub (we have tested with GitHub):
 
 ```
-> code tunnel
+> code tunnel --name=$CLUSTER_NAME-tunnel
```

Comment:
I noticed this - it was calling all connections `daint-ln002` by default, for example.